static/flash.cpp

```diff
@@ -63,37 +66,54 @@ void HostActivation::flashWrite()
     log<level::DEBUG>("HostActivation::flashWrite");
     fs::path uploadDir(IMG_UPLOAD_DIR);
     // CPLD may update via this function
-    for (auto& bmcImage : phosphor::software::image::bmcImages)
+    if (fs::exists(uploadDir / versionId / BIOS_IMAGE))
     {
-        if(bmcImage.compare("image-bios")==0)
+        fs::copy_file(uploadDir / versionId / BIOS_IMAGE, "/tmp/image-bios",
+                          fs::copy_options::overwrite_existing);
```

Comment:
maybe just like above bmcImage use toPath / bmcImage, you can use toPath / BIOS_IMAGE

Reply:
Done